### DISCUSSION TITLE
blank page after clicking on a value to display the graph

### DISCUSSION THREAD

**Discussion by pmosf** [Gulivertx/cbatmo#60] [General]
Hello,
when I click on a value to plot it on the graph, everything disappears, only remains the background image.
Any idea to solve this issue?
By the way, is there a way to recover from this state ? (reset the server to a default state)

Thanks for your support.

**Comment by Gulivertx** [comment]
Hello,

Ho strange, could you please tell me for which module and value you get this, is it for all values, only some ?
To recover the application state juste refresh your browser page.

**Comment by pmosf** [comment]
Hello,
only the values from internal modules are affected. Plots from external modules work fine.
Refreshing the page does not help unfortunately. I have to clear all the browsing data with firefox-esr. With chromium, clearing the data is not enough and I don't manage to recover.

**Comment by Gulivertx** [comment]
yes correct, you have to delete some keys in the browser local storage before to refresh the page.
Delete these keys :
- selected_module
- selected_types

I tried on my installation version 2.3.5 and do not have any problem.

Could you open de dev tools to have console access, then try again, you should have javascript errors, please if you can make a screenshot of the error post it here.

**Comment by pmosf** [comment]
below a picture of the errors (sorry, could not take a screenshot).

![20210104_211449](https://user-images.githubusercontent.com/6083334/103580929-5a04cb00-4edb-11eb-962e-d2ffb39032e6.jpg)

**Comment by Gulivertx** [comment]
ok could you do the same but in `development mode? please, unfortunately this not help.
